docs/Installation.md

```diff
@@ -554,23 +599,24 @@ Reset root password in MySQL (required by MySQL). Replace
 ALTER USER 'root'@'localhost' IDENTIFIED BY '<selected root password>';
 ```
 
-Logout from database:
+Unless you keep an old database, initialize the database:
+```sql
+CREATE DATABASE zonemaster;
+```
 
+Create a new user and give it all permissions on the newly created database:
 ```sql
-exit;
+CREATE USER 'zonemaster'@'localhost' IDENTIFIED BY 'zonemaster';
+GRANT ALL ON zonemaster.* TO 'zonemaster'@'localhost';
 ```
 
-Unless you keep an old database, initialize the database (and give the
-root password when prompted):
+Logout from database:
 
-```sh
-cd `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")'`
-mysql -u root -p < ./initial-mysql.sql
+```sql
+exit;
 ```
 
-> **Note:** This creates a database called `zonemaster`, as well as a user
-> called "zonemaster" with the password "zonemaster" (as stated in the config
-> file). This user has just enough permissions to run the backend software.
+Then update the `/etc/zonemaster/backend_config.ini` file accordingly.

```

Comment:
What is the expected update? The above setting is the default value, isn't it? Should it be
```
Then update the `/etc/zonemaster/backend_config.ini` file with username and password unless the default values are used.
```

Reply:
done